test/test_torch.py

```diff
@@ -8053,28 +8053,30 @@ def test_complex_rot90(self, device, dtype):
             np_fn = partial(np.rot90, k=rot_times, axes=[0, 1])
             self.compare_with_numpy(torch_fn, np_fn, data)
 
+    @unittest.skipIf(not TEST_SCIPY, "Scipy not found")
```

Comment:
Does the OSS CI have SciPy? This test could also use NumPy's `kaiser_window`.

Reply:
I believe that it does have Scipy, the tests don't get skipped, from what I can see. I won't be able to test periodic behavior with numpy (only symmetric), so I'll probably stick with scipy.